<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkFlow</class>
 <widget class="QWidget" name="WorkFlow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="mainSpliiter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="orderLayout" stretch="0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="viewStartButton">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>принятые</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="viewPrintButton">
           <property name="text">
            <string>в печати</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="viewPostPrintButton">
           <property name="text">
            <string>в постпечати</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="readyButton">
           <property name="text">
            <string>готовые</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="viewAllButton">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>весь процесс</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitterOrders">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="OrderStart" name="start" native="true">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_4">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="TasksPWidget" name="print" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_6">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="TasksPWidget" name="postPrint" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_5">
          <layout class="QVBoxLayout" name="readyLayout">
           <item>
            <widget class="OrderReady" name="ready" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="infoLayout" stretch="0">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="InfoWidget" name="info" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OrderStart</class>
   <extends>QWidget</extends>
   <header>widgets/order/orderstart.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TasksPWidget</class>
   <extends>QWidget</extends>
   <header>widgets/order/taskspwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OrderReady</class>
   <extends>QWidget</extends>
   <header>widgets/order/orderready.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InfoWidget</class>
   <extends>QWidget</extends>
   <header>widgets/order/infowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
